lib/command.js

```diff
@@ -735,12 +740,22 @@ Expecting one of '${allowedValues.join("', '")}'`);
    */
   passThroughOptions(passThrough = true) {
     this._passThroughOptions = !!passThrough;
-    if (!!this.parent && passThrough && !this.parent._enablePositionalOptions) {
-      throw new Error('passThroughOptions can not be used without turning on enablePositionalOptions for parent command(s)');
-    }
+    this._checkForIllegalPassThroughOptions(this, this.parent);
     return this;
   }
 
+  /**
+   * @param {Command} command
+   * @param {Command | null} parent
+   * @api private
+   */
+
+  _checkForIllegalPassThroughOptions(command, parent) {
```

Comment:
The high level check is that the settings are likely to work together. `Illegal` is too strong for my liking, and it is the combination rather than just `PassThroughOptions`. Suggest: `_checkForBrokenPassThrough`.

Reply:
Resolved in [43d9faa](https://github.com/tj/commander.js/pull/1937/commits/43d9faa402345f0424ee642203535b2c1aa25bb2).